benchmarks/smallmatvec/manual_versions/libxsmm/submitRun.lsf

```diff
@@ -0,0 +1,12 @@
+#BSUB -J runLFRic
+#BSUB -o /lustre/scafellpike/local/HCP055/pxs02/rxf70-pxs02/PSycloneBench/benchmarks/smallmatvec/manual_versions/libxsmm/%J.out
```

Comment:
Again, I don't think having personal details in this file is a good idea. (Bit late now it's on github though :-)).
Please add a brief comment describing the purpose of the file.

Reply:
Removed. There is information in the README. Is that enough?